bootstrap/test/cache.test.ts

```diff
@@ -1,19 +1,27 @@
-const { expect } = require('chai')
-const { useFakeTimers } = require('sinon')
-const { withCache, defaultOptions } = require('../lib/cache')
-const { LocalLRUCache } = require('../lib/cache/local')
-
-const callAndExpect = async (fn, n, result) => {
+import { expect } from 'chai'
+import { useFakeTimers } from 'sinon'
+import { withCache, ImplCacheOptions, defaultOptions } from '../src/lib/cache'
+import { LocalLRUCache } from '../src/lib/cache/local'
+import { CacheOptions } from '../src/lib/cache'
+import { ExecuteWrappedResponse } from '@chainlink/types'
+
+const callAndExpect = async (fn: Function, n: number, result: any) => {
   while (n--) {
     const { data } = await fn(0)
-    if (n === 0) expect(data).to.equal(result)
+    if (n === 0) expect(data.result).to.equal(result)
   }
 }
 
 // Helper test function: a stateful counter
-const counterFrom = (i = 0) => () => ({ statusCode: 200, data: i++ })
+const counterFrom = (i = 0): ExecuteWrappedResponse => async (request) => {
```

Comment:
Could we revert this part? We are just testing the cache caching the fn input ->  output, there is no need to couple/mix our adapters here or their complicated types 😅